karma-test-shim.js

```diff
@@ -4,7 +4,7 @@ Error.stackTraceLimit = 0; // "No stacktrace"" is usually best for app testing.
 // Uncomment to get full stacktrace output. Sometimes helpful, usually not.
 // Error.stackTraceLimit = Infinity; //
 
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000*10;
```

Comment:
This change ended up not being needed. I'm leaving it in just we are consistent across timeouts.